docs/index.rst

```diff
@@ -0,0 +1,93 @@
+.. _topics-index:
+
+============
+itemloaders
+============
+
+``itemloaders`` provide a convenient mechanism for populating data records.
+It's design provide a flexible, efficient and easy mechanism
+for extending and overriding different field parsing rules, either by raw data,
+or by source format (HTML, XML, etc) without becoming a nightmare to maintain.
+
+To install ``itemloaders``, run::
+
+    pip install itemloaders
+
+
+Getting Started with ``itemloaders``
+====================================
+
+To use an Item Loader, you must first instantiate it. You can either
+instantiate it with a dict-like object (`item`) or without one, in
+which case an `item` is automatically instantiated in the Item Loader ``__init__`` method
+using the `item` class specified in the :attr:`ItemLoader.default_item_class`
+attribute.
+
+Then, you start collecting values into the Item Loader, typically using
+CSS or XPath Selectors. You can add more than one value to
+the same item field; the Item Loader will know how to "join" those values later
+using a proper processing function.
```

Comment:
These two paragraphs are a very confusing introduction :) No need to fix it here though, unless you have an idea of how to do it.

Reply:
Created an issue for now
https://github.com/ejulio/itemloaders/issues/7